src/BaseCommand.js

```diff
@@ -21,6 +20,8 @@ class BaseCommand extends Command {
   getAppConfig () {
     if (!this.appConfig) {
       this.appConfig = loadConfig()
+      // add on app.cliConfig
+      this.appConfig.app.cliConfig = this.config
```

Comment:
I feel it's odd having the `cliConfig` inside the app config, should we use`this.appConfig.cli` instead ?

Reply:
Good point - I'll rename